<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>a</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Начало интервала</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Конец интервала</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Случайное z</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Построить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-50000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-50000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-50000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-500.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
